README.rst

```diff
@@ -51,4 +53,30 @@ Usage
     --index-image container-registry.example.com/index/image:latest
     --operator bundle/image:123
     --arch x86_64
+    --skip-quay
+
+Push the created index image to Quay in a remote server and send a UMB message
+::
+
+  $ export QUAY_PASSWORD=quay_password
+  $ export SSH_PASSWORD=ssh_password
+  $ pubtools-iib-add-bundles \
+    --iib-server iibhostname.example.com \
+    --binary-image container-registry.example.com/binary/image:latest \
+    --index-image container-registry.example.com/index/image:latest \
+    --bundle container-registry.example.com/bundle/image:123 \
+    --arch x86_64 \
+    --skip-pulp \
+    --quay-dest-repo quay.io/namespace/repo \
+    --quay-user namespace+robot_account \
```

Comment:
why there's namespace+robot_account? Shouldn't that be only robot_account?

Reply:
This is the format of Quay robot accounts